build.gradle

```diff
@@ -9,11 +9,12 @@ repositories {
 
 dependencies {
 	compile "com.google.code.gson:gson:2.3"
-
+	compile "org.apache.commons:commons-lang3:3.0"
```

Comment:
I'd say that adding a dependency just to check if a string is empty isn't a good idea.